src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/swift/SwiftAPIClient.java

```diff
@@ -176,20 +142,27 @@ protected String generateToken() {
         return null;
     }
 
-    protected void handleTokenUpdate(ScheduledExecutorService handleTokenUpdateExecutor) {
+    private void handleTokenUpdate(ScheduledExecutorService handleTokenUpdateExecutor) {
         LOGGER.debug("Turning on handle token update.");
 
-        handleTokenUpdateExecutor.scheduleWithFixedDelay(new Runnable() {
-                                                             @Override
-                                                             public void run() {
-                                                                 setToken(generateToken());
-                                                             }
-                                                         }, 0, Integer.parseInt(properties.getProperty(SapsPropertiesConstants.FOGBOW_KEYSTONEV3_UPDATE_PERIOD)),
+        handleTokenUpdateExecutor.scheduleWithFixedDelay(() -> setToken(generateToken()), 0, Integer.parseInt(properties.getProperty(SapsPropertiesConstants.FOGBOW_KEYSTONEV3_UPDATE_PERIOD)),
```

Comment:
Update it for equal this code [here](https://github.com/ufcg-lsd/saps-engine/blob/6d1c212d9766b8761c569d3e67f6e43fafb87520/src/main/java/org/fogbowcloud/saps/engine/core/archiver/Archiver.java#L66)

Reply:
Done.